### PR TITLE
Bootstrap

### PR DESCRIPTION
**Pull request by Lychtear**
Merge the pull-request-1 branch first

### CODE REVIEW

File: sign-in.html [hunk 1]
```diff
@@ -1,27 +1,52 @@
...
+        </div>
     </nav>
 
-    <form class="form">
-        <label>Username <input type="text" name="username" id="username"></label>
-        <label>Password <input type="password" name="password" id="password"></label>
-        <input class="button" type="submit" value="Submit">
-    </form>
+    <div class="container">
+        <form class="form-signin col-md-4 col-md-offset-4 col-sm-4 col-sm-offset-4">
```
Comment (KarinaBuk96) [inline sign-in.html:36]: "col"s should be wrapped with a "row", which should be wrapped with "container".

File: sign-in.html [hunk 2]
```diff
@@ -1,27 +1,52 @@
...
+            <h2 class="form-signin-heading">Please sign in</h2>
+            <label for="inputEmail" class="sr-only">Email address</label>
+            <input type="email" id="inputEmail" class="form-control" placeholder="Email address" required autofocus>
```
Comment (KarinaBuk96) [inline sign-in.html:39]: you should use the class "input-group" for grouping label + input
and the class "input-group-addon" on the label/span near the input

File: sign-up.html [hunk 3]
```diff
@@ -1,28 +1,49 @@
...
     </nav>
 
-    <form class="form">
-        <label>Username <input type="text" name="username" id="username"></label>
-        <label>Password <input type="password" name="password" id="password"></label>
-        <label>Confirm Password <input type="password" name="confirmPassword" id="confirmPassword"></label>
-        <input class="button" type="submit" value="Submit">
-    </form>
+    <div class="container">
+        <form class="form-signin col-md-4 col-md-offset-4 col-sm-4 col-sm-offset-4">
```
Comment (KarinaBuk96) [inline sign-up.html:36]: same as in "sign-in.html" -> "container" is wrapping "row"s which wrap "col"s

File: sign-up.html [hunk 4]
```diff
@@ -1,28 +1,49 @@
...
+            <h2 class="form-signin-heading">Sign Up</h2>
+            <label for="inputUsername" class="sr-only">Username</label>
+            <input type="username" id="inputUsername" class="form-control" placeholder="Username" required autofocus>
```
Comment (KarinaBuk96) [inline sign-up.html:39]: you should use the class "input-group" for grouping label + input
and the class "input-group-addon" on the label/span near the input

File: index.html [hunk 5]
```diff
@@ -12,26 +12,31 @@
...
+                </li>
+            </ul>
+        </div>
     </nav>
 
-    <div class="mainPage">
-        <label>
+    <div class="container">
+        <div>
             <textarea name="tweetText" id="tweetText" cols="80" rows="3" placeholder="tweet text.."></textarea>
```
Comment (KarinaBuk96) [inline index.html:37]: instead of `<textarea>` you can use `<input>` + Bootstrap grid system + the "form-control" class

**Review by KarinaBuk96 — APPROVED** [review]

File: index.html (hunk 5 above)
Comment (Lychtear) [reply index.html:37]: It looks like the wanted results requires textarea (bootstrap-index.html image)

**Pull request merged**